test/Transforms/memory_banking_attrs.mlir

```diff
@@ -0,0 +1,123 @@
+// RUN: circt-opt %s -memory-banking="factor=3 dimension=1" | FileCheck %s --check-prefix ATTRIBUTE
```

Comment:
Curiously, how come `--check-prefix ATTRIBUTE`?

Reply:
ahh because I was trying to stuff everything in the `memory_banking_multi_dim` test file. It didn't work and I simply just copied from there. Now that we are only testing this example for now, I agree that we do not need prefix checking for splitted cases testing.